stable/concourse/README.md

```diff
@@ -129,14 +129,23 @@ The following tables lists the configurable parameters of the Concourse chart an
 | `web.replicas` | Number of Concourse Web replicas | `1` |
 | `web.resources` | Concourse Web resource requests and limits | `{requests: {cpu: "100m", memory: "128Mi"}}` |
 | `web.service.type` | Concourse Web service type | `NodePort` |
+| `web.service.nodePort.fixed` | Concourse Web NodePort fixed if web.service.type is NodePort | `false`|
+| `web.service.nodePort.atc` | Concourse Web ATC NodePort | `nil` |
```

Comment:
Once [what I've suggested in an another comment](https://github.com/kubernetes/charts/pull/2063/files#r143343633) gets accepted, this would be changed to `web.service.atcNodePort`